src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -267,7 +267,7 @@ private function validateAndDenormalize($currentClass, $attribute, $data, $forma
                     throw new LogicException(sprintf('Cannot denormalize attribute "%s" for class "%s" because injected serializer is not a denormalizer', $attribute, $class));
                 }
 
-                if ($this->serializer->supportsDenormalization($data, $class, $format)) {
+                if ($this->serializer->supportsDenormalization($data, $class, $format, $context)) {
```

Comment:
@dunglas Can you check that my conflict resolution is the right one. If not, can you fix it directly on master? Thanks.